bin/program.py

```diff
@@ -29,14 +29,20 @@
     run: '{run}'
 '''
 
+SANITIZER_FLAGS = '''
+cpp:
+    compile: -fsanitize=undefined
```

Comment:
wanna add `,address` also here?

Reply:
i get an error if i add this...
```
Exited with code -6:
==618896==ERROR: AddressSanitizer failed to allocate 0xdfff0001000 (15392894357504) bytes at address 2008fff7000 (errno: 12)
==618896==ReserveShadowMemoryRange failed while trying to map 0xdfff0001000 bytes. Perhaps you're using ulimit -v
```

Reply:
ah yes, `ulimit` / memory limit should be disabled for that. But that probably makes sense to do right?

Reply:
The issue seems to be that we set a memory limit. If we want to have `-fsanitize=address` we have to skip the memory limit. I can add that if you want?